cms/templates/cms/home_page.html

```diff
@@ -67,23 +66,24 @@
           </div>
 
           <h5 class="upper-case">Online Learning</h5>
-          <p>Take graduate-level courses remotely, from anywhere, and meet other students from around the world.</p>
+          <p>Take MIT graduate-level courses remotely, from anywhere, and meet
```

Comment:
Change the text to 
"Take MIT graduate-level courses remotely, from anywhere in the world."